packages/eslint-plugin/package.json

```diff
@@ -25,32 +25,32 @@
   },
   "homepage": "https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/README.md",
   "dependencies": {
-    "@babel/eslint-parser": "^7.16.3",
-    "@babel/eslint-plugin": "^7.14.5",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
+    "@babel/eslint-parser": "^7.19.0",
+    "@babel/eslint-plugin": "^7.19.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.2",
+    "@typescript-eslint/parser": "^5.48.2",
     "change-case": "^4.1.2",
     "common-tags": "^1.8.2",
     "doctrine": "^2.1.0",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-prettier": "^8.6.0",
     "eslint-module-utils": "^2.7.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^25.3.0",
+    "eslint-plugin-import": "^2.27.0",
+    "eslint-plugin-jest": "^26.0.0",
     "eslint-plugin-jest-formatting": "^3.1.0",
-    "eslint-plugin-jsx-a11y": "^6.5.0",
+    "eslint-plugin-jsx-a11y": "^6.7.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.1.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.30.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "eslint-plugin-sort-class-members": "^1.14.0",
+    "eslint-plugin-prettier": "^4.2.0",
+    "eslint-plugin-promise": "^6.1.0",
+    "eslint-plugin-react": "^7.32.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-sort-class-members": "^1.16.0",
     "jsx-ast-utils": "^3.2.1",
     "pkg-dir": "^5.0.0",
     "pluralize": "^8.0.0"
   },
   "peerDependencies": {
-    "eslint": "^8.3.0"
+    "eslint": "^8.32.0"
```

Comment:
Changing the peer dependency on eslint is a breaking change. What package causes this bump and can it be reverted so that we don't force consumers to upgrade more than required?

Reply:
It [needs](https://github.com/typescript-eslint/typescript-eslint/compare/v5.44.0...v5.45.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R82) to be`^8.15.0`. Happy to match that requirement if you judge is better 👍